README.md

```diff
@@ -189,6 +189,8 @@ func collectionSkeletonView(_ skeletonView: UITableView, numberOfRowsInSection s
 // It calculates how many cells need to populate whole tableview
 ```
 
+If you return SkeletonCollectionDataSource.automaticNumberOfRows in the above method, it acts like the default behavior (i.e. it calculates how many cells needed to populate the whole tableview).
```

Comment:
I think this message is important and we should highlighted. 

```suggestion
> 📣 **IMPORTANT!** 
>
> If you return `SkeletonCollectionDataSource.automaticNumberOfRows` in the above method, it acts like the default behavior (i.e. it calculates how many cells needed to populate the whole tableview).
```

Reply:
Agreed.  👍